### PR TITLE
added first task with licking kitty

### PR DESCRIPTION
**Pull request by dmitrysenkovich**

### CODE REVIEW

File: 2015/dmitrysenkovich/task1/test/test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "../src/gif_animation.rb"
+
+def gif_animation_test()
```
Comment (Saicheg) [inline 2015/dmitrysenkovich/task1/test/test.rb]: why do you need this method here?

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    FileUtils.rm_rf(FRAMES_DIR)
+
+    (0..10).each do
+        (0...frames_count).each do |i|
+            puts frames_in_ascii[i]
+            sleep(0.1)
+            puts("\e[H\e[2J")
+        end
+    end
+    system("clear")
```
Comment (Saicheg) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: system "clear" works really slow b/c of creating copy of process. there is muck more elegant way of clear screen.

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require "fileutils"
+
+require "asciiart"
+
+FRAMES_DIR="frames/"
+FRAME_NAME="frame%i.jpg"
+
+def animate_gif(gif_filename)
```
Comment (Saicheg) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: please fix indentation here

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    (0..10).each do
+        (0...frames_count).each do |i|
+            puts frames_in_ascii[i]
+            sleep(0.1)
+            puts("\e[H\e[2J")
+        end
+    end
+    system("clear")
+end
```
Comment (Saicheg) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: EOF should end with \n, please do propriate setup on your IDE

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb (hunk 3 above)
Comment (dmitrysenkovich) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: i wanted to divide system and side libraries

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb (hunk 4 above)
Comment (dmitrysenkovich) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: will correct

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb (hunk 2 above)
Comment (dmitrysenkovich) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: will correct

File: 2015/dmitrysenkovich/task1/test/test.rb (hunk 1 above)
Comment (dmitrysenkovich) [inline 2015/dmitrysenkovich/task1/test/test.rb]: wanted to structure this mini-project better

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb (hunk 4 above)
Comment (dmitrysenkovich) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: i'm working in sulbime3

File: 2015/dmitrysenkovich/task1/src/gif_animation.rb (hunk 3 above)
Comment (dmitrysenkovich) [inline 2015/dmitrysenkovich/task1/src/gif_animation.rb]: but will correct

File: .hound.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+ruby:
```
Comment (Saicheg) [inline .hound.yml]: you don't need this file under Source Control :(

**Pull request closed without merging**